src/app/global.js

```diff
@@ -51,5 +51,7 @@ var
     // Torrent engines
     peerflix = require('peerflix'),
 
+    WebTorrent = require('webtorrent');
+
```

Comment:
Added those requires following the project pattern, the same way as peerflix definition above